test/ui-testing/orgs.js

```diff
@@ -150,20 +145,6 @@ module.exports.test = (uiTestCtx) => {
         });
       });
 
-      if (licensor) {
```

Comment:
Can you keep a test like this? It's important to ensure that the automatic raising-up of a Licensor doesn't break.